### PR TITLE
Add spelling corrections for "construct" variants.

### PR DESCRIPTION
**Pull request by cfi-gb**
See https://github.com/search?q=consrtuct&type=Code

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -17826,6 +17831,7 @@ varients->variants
 varierty->variety
 variey->variety
 varified->verified
+varify->verify
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Seems this is also a real word:
https://www.dictionary.com/browse/varify

Comment (cfi-gb) [reply codespell_lib/data/dictionary.txt]: Mhhh, neither Mozilla/Firefox nor LibreOffice spelling corrections are accepting this as a correct word.

There seems to be also already a "varifying->verifying" in the dictionary.txt one line below. 

On the other hand https://en.wiktionary.org/wiki/varify is available as well, so any opinions to remove varifying->verifying instead?

Comment (peternewman) [reply codespell_lib/data/dictionary.txt]: Probably also `varified->verified`.

I accept it's a rarely used word, but it is a valid word. See also discussion in #712 .

Comment (cfi-gb) [reply codespell_lib/data/dictionary.txt]: I have removed the ``varify->verify`` from this PR again and created a separate PR #1127 for the other variants.

**Pull request merged**